code/player/Inventory.Ammo.cs

```diff
@@ -9,12 +9,12 @@ namespace TTTReborn.Player
 {
     public partial class AmmoInventory
     {
-        private Dictionary<string, int> AmmoList { get; set; } = new();
-        private Inventory Inventory;
+        private Dictionary<string, int> AmmoList { get; } = new();
+        private readonly TTTPlayer _owner;
```

Comment:
This can be a public var imo, should never change and in that way it could be used at other positions as well if needed :)

Reply:
Two notes...

1. I think we should avoid changing access modifiers for use cases that we don't fully know yet. Whenever access to this is needed, someone can just change it to public.
2. The object relationship within this file has a bit of a code smell. It seems a bit odd that `Inventory` would contain a reference to the `player`, since `player` also has a reference to the base `Inventory`. Another reason why I'd leave it private, this should probably be refactored at some point.